posthog/models/error_tracking/error_tracking.py

```diff
@@ -58,3 +58,50 @@ def convert_fingerprints_to_tuples(fps: list[list[str]]):
         # converting back to list of lists before saving
         self.merged_fingerprints = [list(f) for f in merged_fingerprints]
         self.save()
+
+
+class ErrorTrackingSymbolSet(UUIDModel):
```

Comment:
Why did these need to be UUID models again? I forget

Reply:
Multiple teams can have the same symbol set, if they e.g. pull from a shared library or something.